content/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Menggunakan kubectl untuk membuat Deployment
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Belajar tentang aplikasi Deployment.</li>
+                    <li><i>Deploy</i> aplikasi pertama kamu di Kubernetes dengan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3><i>Deployment</i> Kubernetes</h3>
+                <p>
+                Ketika klaster Kubernetes kamu sudah berjalan, kamu dapat men-<i>deploy</i> aplikasi terkontainerisasi di atasnya.
+                Untuk melakukannya, kamu perlu membuat konfigurasi <b>Deployment</b>. Deployemnt menginstruksikan Kubernetes
+                tentang bagaimana cara membuat dan memperbaharui <i>instance</i> aplikasi kamu. Ketika kamu selesai membuat Deployment, Kubernetes master
```

Comment:
https://kbbi.kemdikbud.go.id/entri/memperbarui
```suggestion
                tentang bagaimana cara membuat dan memperbarui <i>instance</i> aplikasi kamu. Ketika kamu selesai membuat Deployment, Kubernetes master
```